doc/classes/ImageTexture.xml

```diff
@@ -4,10 +4,38 @@
 		A [Texture2D] based on an [Image].
 	</brief_description>
 	<description>
-		A [Texture2D] based on an [Image]. Can be created from an [Image] with [method create_from_image].
-		[b]Note:[/b] The maximum image size is 16384×16384 pixels due to graphics hardware limitations. Larger images will fail to import.
+		A [Texture2D] based on an [Image]. For an image to be displayed, an [ImageTexture] has to be created from it using the [method create_from_image] method:
+		[codeblock]
+		var texture = ImageTexture.new()
+		var image = Image.new()
+		image.load("res://icon.png")
+		texture.create_from_image(image)
+		$Sprite2D.texture = texture
+		[/codeblock]
+		This way, textures can be created at run-time by loading images both from within the editor and externally.
+		[b]Warning:[/b] Prefer to load imported textures over loading them from within the filesystem dynamically, as it may not work in exported projects:
```

Comment:
I got confused by this as the different between `Prefer to load imported textures over loading them from within the filesystem dynamically` is not immediately obvious.

I'd suggest maybe:
```suggestion
		[b]Warning:[/b] Prefer to load imported textures with [method @GDScript.load] over loading them from within the filesystem dynamically with [method Image.load], as it may not work in exported projects:
```